### ISSUE TITLE
[request] fluidsynth/2.3.6

### ISSUE THREAD

**Issue by wouterbeek** [conan-io/conan-center-index#15397]
### Package Name/Version

fluidsynth/2.3.6

### Webpage

https://www.fluidsynth.org/

### Source code

https://github.com/FluidSynth/fluidsynth

### Description of the library/tool

FluidSynth is a real-time software synthesizer based on the SoundFont 2 specifications and has reached widespread distribution. FluidSynth itself does not have a graphical user interface, but due to its powerful API several applications utilize it and it has even found its way onto embedded systems and is used in some mobile apps.

**Comment by jwillikers** [comment]
Yes, this would be fantastic. I just started using this recently with a midi keyboard, too.

**Comment by wouterbeek** [comment]
Bincrafters has this package at https://github.com/bincrafters/community/tree/main/recipes/fluidsynth
